From: John Zupin <johnz@lunarg.com>
Date: Fri, 19 Jun 2020 15:22:33 -0600
Subject: update install targets for root CMakeLists.txt file to include/dxc/

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a636cf5..b0d49a9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -714,7 +714,7 @@ add_subdirectory(cmake/modules)
 
 if (NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
   install(DIRECTORY include/llvm include/llvm-c
-    DESTINATION include
+    DESTINATION include/dxc/
     COMPONENT llvm-headers
     FILES_MATCHING
     PATTERN "*.def"
@@ -726,7 +726,7 @@ if (NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
     )
 
   install(DIRECTORY ${LLVM_INCLUDE_DIR}/llvm
-    DESTINATION include
+    DESTINATION include/dxc/
     COMPONENT llvm-headers
     FILES_MATCHING
     PATTERN "*.def"
@@ -750,4 +750,4 @@ endif()
 
 if (NOT "${DXC_CMAKE_ENDS_INCLUDE}" STREQUAL "")
   include(${DXC_CMAKE_ENDS_INCLUDE})
-endif()
\ No newline at end of file
+endif()
