From: John Zupin <johnz@lunarg.com>
Date: Fri, 19 Jun 2020 15:22:33 -0600
Subject: update install targets for root CMakeLists.txt file to include/dxc/

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 823cad3..d344013 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -734,7 +734,7 @@ add_subdirectory(cmake/modules)
 
 if (NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
   install(DIRECTORY include/llvm include/llvm-c
-    DESTINATION include
+    DESTINATION include/dxc/
     COMPONENT llvm-headers
     FILES_MATCHING
     PATTERN "*.def"
@@ -746,7 +746,7 @@ if (NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
     )
 
   install(DIRECTORY ${LLVM_INCLUDE_DIR}/llvm
-    DESTINATION include
+    DESTINATION include/dxc/
     COMPONENT llvm-headers
     FILES_MATCHING
     PATTERN "*.def"
