From: John Zupin <johnz@lunarg.com>
Date: Fri, 19 Jun 2020 15:50:10 -0600
Subject: update install targest for llvm files

---
 cmake/modules/AddLLVM.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/modules/AddLLVM.cmake b/cmake/modules/AddLLVM.cmake
index d600ec7..2578bb0 100644
--- a/cmake/modules/AddLLVM.cmake
+++ b/cmake/modules/AddLLVM.cmake
@@ -553,7 +553,7 @@ macro(add_llvm_library name)
 
       install(TARGETS ${name}
             EXPORT LLVMExports
-            ${install_type} DESTINATION lib${LLVM_LIBDIR_SUFFIX}
+            ${install_type} DESTINATION lib${LLVM_LIBDIR_SUFFIX}/dxc/
             COMPONENT ${name})
 
       if (NOT CMAKE_CONFIGURATION_TYPES)
@@ -583,12 +583,12 @@ macro(add_llvm_loadable_module name)
           # DLL platform
           set(dlldir "bin")
         else()
-          set(dlldir "lib${LLVM_LIBDIR_SUFFIX}")
+          set(dlldir "lib${LLVM_LIBDIR_SUFFIX}/dxc/")
         endif()
         install(TARGETS ${name}
           EXPORT LLVMExports
           LIBRARY DESTINATION ${dlldir}
-          ARCHIVE DESTINATION lib${LLVM_LIBDIR_SUFFIX})
+          ARCHIVE DESTINATION lib${LLVM_LIBDIR_SUFFIX}/dxc/)
       endif()
       set_property(GLOBAL APPEND PROPERTY LLVM_EXPORTS ${name})
     endif()
