From: John Zupin <johnz@lunarg.com>
Date: Tue, 27 Apr 2021 18:29:49 -0600
Subject: fix override error

---
 tools/clang/tools/dxcompiler/dxcompilerobj.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/clang/tools/dxcompiler/dxcompilerobj.cpp b/tools/clang/tools/dxcompiler/dxcompilerobj.cpp
index ae40496..a04ade5 100644
--- a/tools/clang/tools/dxcompiler/dxcompilerobj.cpp
+++ b/tools/clang/tools/dxcompiler/dxcompilerobj.cpp
@@ -552,6 +552,8 @@ class DxcCompiler : public IDxcCompiler3,
                     public IDxcVersionInfo3,
 #ifdef SUPPORT_QUERY_GIT_COMMIT_INFO
                     public IDxcVersionInfo2
+#else
+                    public IDxcVersionInfo
 #endif // SUPPORT_QUERY_GIT_COMMIT_INFO
 {
 private:
