From: John Zupin <johnz@lunarg.com>
Date: Tue, 16 Nov 2021 17:37:16 -0700
Subject: fix python script when running make

---
 utils/hct/hctgen.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/utils/hct/hctgen.py b/utils/hct/hctgen.py
index 61da281..e7fd857 100755
--- a/utils/hct/hctgen.py
+++ b/utils/hct/hctgen.py
@@ -51,6 +51,8 @@ def writeCodeTag(args):
   return 0
   
 def openOutput(args):
+  if not os.path.isdir(os.path.dirname(args.output)):
+    os.mkdir(os.path.dirname(args.output))
   return open(args.output, 'w', newline=getNewline(args))
 
 def printHeader(out, filename):
